cmd/spire-server/cli/bundle/common.go

```diff
@@ -211,3 +216,66 @@ func printBundle(out io.Writer, bundle *common.Bundle, header bool) error {
 
 	return nil
 }
+
+func printRegistrationBundle(out io.Writer, bundle *registration.Bundle, format string, header bool) error {
+	if bundle == nil {
+		return errors.New("no bundle provided")
+	}
+
+	format, err := validateFormat(format)
+	if err != nil {
+		return err
+	}
+
+	if header {
```

Comment:
it is never true

Reply:
This function is only used by the `bundle show` command, so it shouldn't really be part of this common file. I'll move it to show.go and remove the header parameter.